models/comments.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection")
+exports.removeCommentById = (commentId) => {
+    return db
+    .query('DELETE FROM comments WHERE comment_id = $1 RETURNING*', [commentId])
+    .then(({rows}) => {
+        if(rows.length === 0){
```

Comment:
rows.length will be 0 if the comment exist and has been deleted 'cause delete doesn't return anything. Using rowCount rather than rows.length will be more accurate